projects/plugins/jetpack/modules/contact-form.php

```diff
@@ -35,24 +35,6 @@
 	return true; // Not returning true will cause the module to become deactivated.
 }
 
-require_once __DIR__ . '/contact-form/grunion-contact-form.php';
-
-/*
- * Filters if the new Contact Form Editor View should be used.
- *
- * A temporary filter to disable the new Editor View for the older UI.
- * Please note this filter and the old UI will be removed in the future.
- * Expected to be removed in Jetpack 5.8 or if a security issue merits removing the old code sooner.
- *
- * @since 5.2.0
- * @deprecated 13.2.0
- *
- * @param boolean $view Use new Editor View. Default true.
- */
-if ( is_admin() && apply_filters_deprecated( 'tmp_grunion_allow_editor_view', array( true ), '13.2.0', '', 'This functionality will be removed in an upcoming version.' ) ) {
-	require_once __DIR__ . '/contact-form/grunion-editor-view.php';
-}
-
 /**
  * Register Jetpack Form patterns
```

Comment:
Gotcha.

Reply:
The code that registers the patterns, `jetpack_form_register_pattern`. It's also in `Utils::register_pattern()`

Reply:
Ah right, I was busy searching for the function name instead. So since it lives elsewhere now with a different function name, presumably I'd still need to deprecate this specific function initially first before deleting it, similarly to the Google Drive Helper functions?

Reply:
Yes, I think that's the right thing to do! 👍